evap/student/templates/student_vote.html

```diff
@@ -43,6 +40,35 @@ <h3 class="mb-3">{{ course.name }} ({{ course.semester.name }})</h3>
     <form id="student-vote-form" method="POST" class="form-horizontal">
         {% csrf_token %}
 
+        {% if participants_warning %}
+            <div class="card card-outline-warning mb-3">
+                <div class="card-header">
+                    <span class="fas fa-exclamation-triangle"></span> {% trans 'Small number of participants' %}
+                </div>
+                <div class="card-body">
+                    {% blocktrans %}
+                        Only a small number of people can take part in this evaluation. You should be aware that
+                        contributors might be able to guess who voted for a specific answer and who wrote a text
+                        answer. Results will only be published, if two or more people participated in the evaluation.
+                    {% endblocktrans %}
+                    {% if course.num_voters == 0 %}
+                        <br />
+                        {% blocktrans %}
+                            If you want your text answers to be shown to the contributors even if you remain the only
```

Comment:
maybe prepend "You're the first person taking part in this evaluation." to give an explicit answer to "why am i seeing this"?